.rubocop.yml

```diff
@@ -0,0 +1,508 @@
+require: rubocop-rspec
```

Comment:
This seems to have diverged quite a bit from the one discussed and implemented[1] in the module tool. Are there any docs or discussions on puppet-users that go over this?

It seems that it is explicitly enabling cops as opposed to enabling all of them and then ignoring some. Is that true, is it just being really explicit or are these extras not enabled by default?

[1] - https://github.com/puppetlabs/puppet/blob/master/lib/puppet/module_tool/skeleton/templates/generator/.rubocop.yml